src/components/common/LogPage/index.tsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
 import * as gtag from 'lib/gtag';
+import uuidv4 from 'utils/ts/uuidGenerater';
+import { UserResponse } from 'api/auth/entity';
 import { useLocation } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { userInfoState } from 'utils/recoil/userInfoState';
+
+const userUniqueIdGenerator = (userInfo: UserResponse | null | undefined) => {
+  const uuid = uuidv4();
+  if (userInfo?.student_number && userInfo.student_number.length > 0) {
```

Comment:
undefined와의 대소비교가 false가 반환되니까 로직을 합쳐도 괜찮지 않을까요!?

Reply:
로직을 합친다는게 해당 코드 식으로 말씀하신 것 같습니다.
```suggestion
  if (userInfo?student_number) {
```
제가 string에 대해서 ''값을 생각해서 해당 방식을 썼었던 것 같습니다. 
다시 생각해보니 js에서는 이게 false이니 조건문을 위와 같이 하는게 깔끔할 것 같습니다. 감사합니다.

Reply:
수정 커밋입니다
[3101e9a](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/111/commits/3101e9ada5fa89a9872d08e479516ef5c6f08243)